qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -425,46 +425,67 @@ def resource_estimate(
 
     def submit_aces(
         self,
-        qubits: qiskit.QuantumRegister,
+        qubits: List[int],
         shots: int,
         num_circuits: int,
         mirror_depth: int,
         extra_depth: int,
-        **kwargs: Any,
+        method: Optional[str] = None,
+        noise: Optional[str] = None,
+        error_prob: Optional[Union[float, Tuple[float, float, float]]] = None,
+        tag: Optional[str] = None,
+        lifespan: Optional[int] = None,
     ) -> str:
-        """Submits the jobs to characterize this target through the ACES protocol.
+        """Performs a POST request on the `/aces` endpoint.
```

Comment:
the old docstring seems ok here? maybe this should be copied to `css.Service` and `gss.Services` (ideally with a reference to the ACES paper as well)